model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -57,7 +57,7 @@ data class AnalyzerResult(
      * A map that holds arbitrary data. Can be used by third-party tools to add custom data to the model.
```

Comment:
commit msg: What's the reason for not taking it into account ?

Reply:
See https://github.com/heremaps/oss-review-toolkit/pull/1394/files#diff-472e0854539fa2448a66c7b5f18ce2d6R84, we usually don't want that. Please note this is a draft PR. I'd like to discuss this in person with you and @mnonnenmacher next week.